### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by 94noni**
Some improvements

- front (css/html)
- back (more readable)
- fix var name
- add toggleAll columns feature
- separate routing file into 2

### CODE REVIEW

File: Util/Doctrine/SingleColumnArrayHydrator.php [hunk 1]
```diff
@@ -22,11 +22,10 @@ protected function hydrateAllData()
             $value = $data[0];
 
             if(is_numeric($value)) {
-                if (false === mb_strpos($value, '.', 0, 'UTF-8')) {
-                    $value = (int) $value;
-                } else {
-                    $value = (float) $value;
-                }
+                $value = (false === mb_strpos($value, '.', 0, 'UTF-8'))
```
Comment (kuczek) [inline Util/Doctrine/SingleColumnArrayHydrator.php]: @94noni It's less readable than previous version....

File: Translation/Exporter/ExporterCollector.php [hunk 2]
```diff
@@ -25,28 +33,23 @@ public function addExporter($id, ExporterInterface $exporter)
...
-     * @param string $format
-     * @return ExporterInterface
+     * @param $format
+     * @param $file
+     * @param $translations
+     * @return bool
+     * @throws \RuntimeException
      */
-    public function getByFormat($format)
+    public function export($format, $file, $translations)
```
Comment (kuczek) [inline Translation/Exporter/ExporterCollector.php]: @94noni This is BC break.

Comment (94noni) [inline Translation/Exporter/ExporterCollector.php]: @kuczek are devs really using (or going to use) this service method in their src?

File: Util/Doctrine/SingleColumnArrayHydrator.php (hunk 1 above)
Comment (94noni) [inline Util/Doctrine/SingleColumnArrayHydrator.php]: @kuczek well i think ternary operator is more readable :s

File: Translation/Exporter/ExporterCollector.php (hunk 2 above)
Comment (kuczek) [inline Translation/Exporter/ExporterCollector.php]: @94noni You never know. But if this should be merged to this version, there should not be something like that, you can create method export, that you should leave the old one (maybe with @deprecated).

File: Resources/config/routing.yml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 lexik_translation_list:
-    pattern:      /translations
+    pattern:      /list
     defaults:     { _controller: "LexikTranslationBundle:Rest:list" }
     requirements: { _method: GET }
 
 lexik_translation_update:
-    pattern:      /translations/{id}
+    pattern:      /update/{id}
```
Comment (cedric-g) [inline Resources/config/routing.yml]: These two routes aim to be restful, so the url is about what you deal with (translations) and the http method is about what you want to do.
Maybe I should put them in a separated file to make the difference between API and other routes.

File: Storage/AbstractDoctrineStorage.php [hunk 4]
```diff
@@ -12,6 +12,13 @@
 abstract class AbstractDoctrineStorage implements StorageInterface
 {
     /**
+     * All the available config storage types.
+     */
+    const STORAGE_ORM     = 'orm';
+    const STORAGE_MONGODB = 'mongodb';
+    const STORAGE_PROPEL  = 'propel';
```
Comment (cedric-g) [inline Storage/AbstractDoctrineStorage.php]: You should move the constants into `StorageInterface`. Propel is not a part of Doctrine so it's a bit weird to call `AbstractDoctrineStorage::STORAGE_PROPEL`.

File: Translation/Exporter/ExporterCollector.php [hunk 5]
```diff
@@ -27,26 +35,36 @@ public function addExporter($id, ExporterInterface $exporter)
     /**
      * Returns an exporter that support the given format.
      *
-     * @param string $format
-     * @return ExporterInterface
+     * @param $format
+     * @return mixed
+     * @throws \RuntimeException
      */
```
Comment (cedric-g) [inline Translation/Exporter/ExporterCollector.php]: Please keep the phpdoc
```
@param string $format
@return ExporterInterface
```

File: Resources/config/routing.yml (hunk 3 above)
Comment (94noni) [inline Resources/config/routing.yml]: +1 for separated file then, the primary idea was to be more precise

File: Storage/AbstractDoctrineStorage.php (hunk 4 above)
Comment (94noni) [inline Storage/AbstractDoctrineStorage.php]: :+1: indeed

**Pull request merged**